app/models/participant_declaration.rb

```diff
@@ -26,6 +26,7 @@ class ParticipantDeclaration < ApplicationRecord
            through: :billable_statement_line_items,
            source: :statement
 
+  attribute :state, :string
```

Comment:
Interesting that this is needed. [Background reading](https://github.com/rails/rails/issues/45668) for anyone else who comes across it.